core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/internal/EventArrays.java

```diff
@@ -4,15 +4,18 @@
 
```

Comment:
no copyright on this file, is it normal?

Reply:
Ooops. This file should definitely have an Oracle Copyright Notice.

Reply:
done